### ISSUE TITLE
Weird output in jobs and slots

### ISSUE THREAD

**Issue by kmanalo** [rlyon/qhost#17]
@rlyon I'm only getting 0's and 1's in jobs and slots, are you getting that? I have a one-line fixer for you, but it depends on the that last pull request regarding the set vs. list deal.  Furthermore, this breaks some of the unit testing, as this changes some of the output (I think I saw 7 tests fail)

Anyways, the pre-parsed output looks like (pulled from one node)

```
'3/5674.generic.edu,4/5674.generic.edu,5/5674.generic.edu,10/5697.generic.edu,11/5697.generic.edu,12/5697.generic.edu,13/5697.generic.edu,14/5697.generic.edu,15/5697.generic.edu,16/5697.generic.edu,17/5697.generic.edu,6/5747.generic.edu,7/5747.generic.edu,8/5747.generic.edu,18/5747.generic.edu,9/7358.generic.edu,19/7358.generic.edu,0/9041.generic.edu,1/9041.generic.edu,2/9041.generic.edu'
```
Is this how your pbsnodes output appears also?

**Comment by kmanalo** [comment]
Hmm, I noticed a different pattern in how the parsing is done with the output examples, I'll look into this further

**Comment by rlyon** [comment]
Fixed with pull #18
